pkg/crypto/rsa.go

```diff
@@ -0,0 +1,53 @@
+package crypto
```

Comment:
should this be called `rsa` or do you plan to add more functions?

Reply:
I plan to add more functions.